.github/workflows/e2e.yml

```diff
@@ -79,7 +79,8 @@ jobs:
             --image-replacements workspace/just.containerlookup \
             --namespace-file workspace/e2e.namespace \
             --platform ${{ inputs.platform }} \
-            --skip-undeploy="${{ inputs.skip-undeploy && 'true' || 'false' }}"
+            --skip-undeploy="${{ inputs.skip-undeploy && 'true' || 'false' }}" \
```

Comment:
fyi, @miampf is removing `--skip-undeploy` in https://github.com/edgelesssys/contrast/pull/1046.